src/mixins/filing-template-mixin.ts

```diff
@@ -345,6 +345,186 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     }
   }
 
+  /**
+   * Builds an continuation in filing from store data. Used when saving a filing.
+   * @returns the filing body to save
+   */
+  buildContinuationInFiling (): ContinuationInFilingIF {
+    function fixNullAddressType (orgPeople: OrgPersonIF[]): OrgPersonIF[] {
+      return orgPeople.map(p => {
+        if (p.deliveryAddress?.addressType === null) delete p.deliveryAddress.addressType
+        if (p.mailingAddress?.addressType === null) delete p.mailingAddress.addressType
+        return p
+      })
+    }
```

Comment:
This is now the 4th or 5th copy of this function. Can you look into pulling it out of the main functions and moving it to the bottom of this file?

Reply:
Now it's a standalone function that all the main functions call. Great catch to minimize code duplication Sev.